shell/platform/android/io/flutter/app/FlutterApplication.java

```diff
@@ -33,4 +35,12 @@ public Activity getCurrentActivity() {
   public void setCurrentActivity(Activity mCurrentActivity) {
     this.mCurrentActivity = mCurrentActivity;
   }
+
+  // This override allows split dynamic feature modules to work.
+  @Override
+  protected void attachBaseContext(Context base) {
+    super.attachBaseContext(base);
+    // Emulates installation of future on demand modules using SplitCompat.
+    SplitCompat.install(this);
```

Comment:
Not measured, but as far as I can tell, there is no discussion online on the overhead of this.

Reply:
FWIW, when time comes for docs, we still need to write this since we've essentially deprecated this class https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/app/android.tmpl/app/src/main/AndroidManifest.xml.tmpl#L3.